app/economy/management/commands/refresh_bounties.py

```diff
@@ -31,20 +35,22 @@ def add_arguments(self, parser):
             help='Pulls remote info about bounty too'
         )
 
-    help = 'refreshes the triggers associated with current bounties'
-
     def handle(self, *args, **options):
+        """Refresh all bounties.
 
-        current_bounties = Bounty.objects.filter(current_bounty=True).all()
-        for b in current_bounties:
-            if options['remote']:
-                b.fetch_issue_description()
-            b.save()
-            print('1/ refreshed {}'.format(b.pk))
+        Attributes:
+            all_bounties (QuerySet of Bounty): The queryset of all Bounties.
+            fetch_remote (bool): Whether or not to fetch remote bounties.
+                Defaults to: `False` unless user passes the remote option.
 
+        """
         all_bounties = Bounty.objects.all()
-        for b in all_bounties:
-            if not b.avatar_url:
-                b.avatar_url = b.get_avatar_url()
-            b.save()
-            print('2/ refreshed {}'.format(b.pk))
+        fetch_remote = options['remote']
+        for bounty in all_bounties:
```

Comment:
unless im mistaken, i think we only want to iterate `current_bounties` here?

Reply:
I'm performing the iteration only once. If it's a current bounty **and** remote is set to true, we fetch the issue.  It performs the same as previously, but only uses one QS versus iterating two and saving twice.

Reply:
ah good call. carry on